cmake/BuildLibCURL.cmake

```diff
@@ -234,12 +221,6 @@ macro(curl_build)
             set(CURL_LIBRARIES ${CURL_LIBRARIES} resolv)
         endif()
     endif()
-    if (BUNDLED_LIBCURL_USE_NGHTTP2)
-        set(CURL_LIBRARIES ${CURL_LIBRARIES} ${NGHTTP2_LIBRARIES})
-    endif()
-    if (TARGET_OS_LINUX OR TARGET_OS_FREEBSD)
-        set(CURL_LIBRARIES ${CURL_LIBRARIES} rt)
-    endif()
```

Comment:
It seems, this block is dropped by a mistake: it is not related to HTTP/2.0 and libnghttp2.